### ISSUE TITLE
Mechs offer evil moth plushies an Act of Contrition for their sins? (forgotten suicide count)

### ISSUE THREAD

**Issue by Learner10** [tgstation/tgstation#77166]
## Round ID:
211167

## Testmerges:
#77030 

## Reproduction:
So in short, a moth plushie (specifically the psychologist's named plushie Dr Moff on Icebox) was used to commit suicide, raising it's kill count to 1. It was examined, and it reported the proper matching description stating it killed someone. 30 seconds later, it was picked up by a Ripley piloted by a positron, where it was in storage for about... 10-20 minutes before being dropped in robotics, still named Dr Moff, but with it's original description, seemingly having forgotten.

**Comment by Sealed101** [comment]
i sense adminbus at play. judging by the logs though this varedit happened after the discovery of the desc reset but still

`[2023-07-27 20:55:14.037] GAME-COMPAT: ADMIN: Bmon/(June Creighton) modified Dr. Moff's suicide_count from 1 to 4`

tl;dr i have zero clue on how could this have happened. needs repro

**Comment by Learner10** [comment]
> i sense adminbus at play. judging by the logs though this varedit happened after the discovery of the desc reset but still
> 
> `[2023-07-27 20:55:14.037] GAME-COMPAT: ADMIN: Bmon/(June Creighton) modified Dr. Moff's suicide_count from 1 to 4`
> 
> tl;dr i have zero clue on how could this have happened. needs repro

In admin chat, it was requested that something be changed to see if it'd fix it. Chaning it STILL visually reported nothing.
